client/src/App.jsx

```diff
@@ -12,10 +12,10 @@ function App() {
       <nav>
         <ul>
           <li>
-            <Link to="/">Home</Link>
-            <Link to="/about">About</Link>
-            <Link to="/contact">Contact</Link>
-            <Link to="/cities">Search</Link>
+            <Link to="/"> Home</Link>
+            <Link to="/about"> About</Link>
+            <Link to="/contact"> Contact</Link>
+            <Link to="/cities"> Search</Link>
```

Comment:
if you need spaces near every link, it would be better to add styling

Reply:
This will be styled once Header and Footer are added.